themes/default/layouts/challenge/list.html

```diff
@@ -0,0 +1,76 @@
+{{ define "hero" }}{{ partial "hero.html" (dict "title" "Challenges") }}{{ end }}
```

Comment:
Is there a way to utilize the super cool hero banner we have. It would make it pop more and make it more exciting

Reply:
We're using it in two different places already. On the individual challenge page (which is what most people will be seeing -- we aren't prioritizing pushing traffic to the /challenge page) and on the announcement blog. I don't want to overuse it and look repetitive.